symengine/expression.h

```diff
@@ -0,0 +1,128 @@
+/**
+ *  \file expression.h
+ *  Includes the Expression class
+ *
+ **/
+
+#ifndef SYMENGINE_EXPRESSION_H
+#define SYMENGINE_EXPRESSION_H
+
+#include <iostream>
+#include <type_traits>
+
+#include <symengine/add.h>
+#include <symengine/basic.h>
+#include <symengine/symengine_rcp.h>
+#include <symengine/integer.h>
+#include <symengine/mul.h>
+#include <symengine/pow.h>
+#include <symengine/symbol.h>
+
+namespace SymEngine
+{
+
+class Expression
+{
+private:
+	RCP<const Basic> m_basic;
```

Comment:
If you use Vim, you should configure it to do 4 spaces (or 2 spaces in some
projects). Here is my config:

https://github.com/certik/dotfiles/blob/bf5d3f006782000441d46d818dd24ada3e4094b1/.vimrc

and you can drop in customized .project.vim into projects that need
different amount of space than 4.

On Tue, Jul 7, 2015 at 9:07 AM, Sumith <notifications@github.com> wrote:

> In symengine/expression.h
> <https://github.com/sympy/symengine/pull/500#discussion_r34042859>:
>
> > +
> > +#include <symengine/add.h>
> > +#include <symengine/basic.h>
> > +#include <symengine/symengine_rcp.h>
> > +#include <symengine/integer.h>
> > +#include <symengine/mul.h>
> > +#include <symengine/pow.h>
> > +#include <symengine/symbol.h>
> > +
> > +namespace SymEngine
> > +{
> > +
> > +class Expression
> > +{
> > +private:
> > +	RCP<const Basic> m_basic;
>
> Will do. Tab on my editor always comes here as 8 spaces [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sympy/symengine/pull/500/files#r34042859>.
>